src/Rules/BinaryDist.hs

```diff
@@ -83,7 +106,7 @@ bindistInstallFiles :: [FilePath]
 bindistInstallFiles =
   [ "config.sub", "config.guess", "install-sh"
   , "mk" -/- "config.mk.in", "mk" -/- "install.mk.in"
-  , "settings.in", "README", "INSTALL"
+  , "mk" -/- "project.mk.in", "settings.in", "README", "INSTALL"
```

Comment:
I don't see any occurence of `project.mk` or `project.mk.in` besides this one, does the code as it is now somehow implicitly requires it? Or something will, in an upcoming patch? Or is it just a leftover from an experiment of yours?

Reply:
project.mk is a dependency of config.mk. I am using config.mk.

Reply:
OK, as long as it doesn't pull anything else with it, that should be fine :-)

Reply:
Yes, project.mk is independent.